src/osemgrep/language_server/server/RPC_server.mli

```diff
@@ -1,3 +1,24 @@
+module type LSIO = sig
```

Comment:
Please comments. I know it's a draft and you're trying to get LSP.js ASAP.
but please add just a few comments trying to impersonate for a few minutes someone who didn't write this
code and has no idea why we need all this stuff.

Reply:
Yes sorry, will go through and add comments